Code/Tools/ProjectManager/Source/AddRemoteTemplateDialog.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <ProjectInfo.h>
+
+#include <QDialog>
+#endif
+
+QT_FORWARD_DECLARE_CLASS(QLabel)
+QT_FORWARD_DECLARE_CLASS(QCheckBox)
+QT_FORWARD_DECLARE_CLASS(QDialogButtonBox)
+QT_FORWARD_DECLARE_CLASS(QPushButton)
+QT_FORWARD_DECLARE_CLASS(QTimer)
+
+namespace O3DE::ProjectManager
+{
+    QT_FORWARD_DECLARE_CLASS(FormLineEditWidget)
+
+    class AddRemoteTemplateDialog
```

Comment:
If we add any more "download remote XYZ" dialogs, we should look into consolidating these into a single dialog were you can set the text, since the actual UI is so similar

Reply:
Already planning on having the design of this to match downloading projects after next release to make this easier to combine.